scripts/ci/create-benchmark-pr.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
```

Comment:
@joao-paulo-parity I know that it's merged, but please add a description what the script does

Reply:
👍 
I did not find a **good** doc standard for bash scripts yet but I ended up using 2 `#` such as `## Some doc` in a bunch of scripts.

That allows extracting the "doc" of a script easily while leaving the comments aside. This is especially useful when using `fzf` as you can use the preview feature to show the doc with a simple grep. This is what the (WIP) `rel` scripts do:

<img width="830" alt="image" src="https://user-images.githubusercontent.com/738724/197166372-3c5d9094-4cd7-4fff-8667-e952fb842d3f.png">

Happy to hear about better options though.